app/src/main/java/org/simple/clinic/newentry/PatientEntryViewEffectHandler.kt

```diff
@@ -53,4 +55,36 @@ class PatientEntryViewEffectHandler(
       showDOBExceedsMinLimitError(false)
     }
   }
+
+  private fun showValidationErrors(errors: List<PatientEntryValidationError>) {
+    errors
+        .onEach { Analytics.reportInputValidationError(it.analyticsName) }
+        .forEach {
+          when (it) {
+            PatientEntryValidationError.FullNameEmpty -> validationActions.showEmptyFullNameError(true)
```

Comment:
Why do we have a different interface for `PatientEntryValidationActions` (UiActions)? It should ideally be included in the `PatientEntryUiActions`. WDYT?

Reply:
Yep, good point. Will make that change.